### DISCUSSION TITLE
IOS 快捷指令剪贴板内容发送至 flomo

### DISCUSSION THREAD

**Discussion by zhxliao** [flomoapp/3rd-party-tools#18] [分享创造]
拷贝如下链接至ios设备，并用safari打开：

https://www.icloud.com/shortcuts/6971062edb0145c7a00572a817e08995

效果如视频所示：

https://user-images.githubusercontent.com/4892002/102979606-418ecb00-4541-11eb-9dea-3eb2ee6ed82d.MOV

**Comment by lightory** [comment]
还配了效果视频，赞
